### PR TITLE
add basic dashboard info for NGINX Plus

### PR DESCRIPTION
**Pull request by ilyam8**
##### Summary

ssia

##### Test Plan

Tested locally by copying dashboard_info.js to a server that collects metrics from NGINX Plus.

##### Additional Information

<details> <summary>For users: How does this change affect me?</summary>
  
</details>

### CODE REVIEW

**Review by thiagoftsm — APPROVED** [review]
@ilyam8 now that  I am approving here, please, take a look at https://github.com/netdata/netdata/pull/14001, because I modified as you required. :smile:

File: web/gui/dashboard_info.js [hunk 1]
```diff
@@ -4418,6 +4418,150 @@ netdataDashboard.context = {
...
+        info: 'The number of client requests that are currently being processed by the HTTP Server Zone.'
+    },
+    'nginxplus.http_server_zone_requests_discarded_rate': {
+        info: 'The number of requests to the HTTP Server Zone completed without sending a response.'
+    },
+    'nginxplus.http_location_zone_requests_rate': {
+        info: 'The number of requests to the HTTP Location Zone.'
+    },
+    'nginxplus.http_location_zone_responses_per_code_class_rate': {
+        info: 'The number of responses from the HTTP Location Zone. Responses grouped by HTTP status code class.'
```
Comment (MrZammler) [inline web/gui/dashboard_info.js:4469]: Maybe just `Responses are grouped by HTTP status code class.`, but no big deal. There is one similar above as well and below, if you decide to change.

File: web/gui/dashboard_info.js [hunk 2]
```diff
@@ -4418,6 +4418,150 @@ netdataDashboard.context = {
...
+        info: 'The current number of HTTP Upstream servers removed from the group but still processing active client connections.'
+    },
+    'nginxplus.stream_upstream_server_connections_rate': {
+        info: 'The number of connections forwarded to the Stream Upstream Server.'
+    },
+    'nginxplus.stream_upstream_server_traffic_rate': {
+        info: 'The amount of traffic transferred to and from the Stream Upstream Server.'
+    },
+    'nginxplus.stream_upstream_server_state': {
+        info: 'The current state of the Stream Upstream Server. Status active if set to 1.'
```
Comment (MrZammler) [inline web/gui/dashboard_info.js:4550]: Maybe also `Status is active if set to 1.`

**Review by MrZammler — APPROVED** [review]

File: web/gui/dashboard_info.js (hunk 1 above)
Comment (ilyam8) [reply web/gui/dashboard_info.js:4469]: Those are specifically for the HTTP Location Zone.

Comment (ilyam8) [reply web/gui/dashboard_info.js:4469]: Ah, you mean _are_ is missing. Probably you are right, but I think it is not an error w/o _are_ there.

Comment (MrZammler) [reply web/gui/dashboard_info.js:4469]: Yes, sorry wasn't clear, just the `are` there seems more "nice" :-)

**Pull request merged**